### PR TITLE
fix(torch-frontend): fix errors in torch.log and torch.erf

### PR DESCRIPTION
**Pull request by yopknopixx**
# PR Description

Fix errors in torch.log and torch.erf with the paddle backend caused by unsupported datatypes

### CODE REVIEW

**Comment by yopknopixx** [comment]
Hey @aparajith21! This is not an open task and is related to the Fixing Priority Functions task.

**Comment by vedpatwardhan** [comment]
Hey @yopknopixx, feel free to request a review once you're done with the changes, thanks!

File: ivy/functional/frontends/torch/pointwise_ops.py [hunk 1]
```diff
@@ -181,7 +181,8 @@ def div(input, other, *, rounding_mode=None, out=None):
         return ivy.divide(input, other, out=out)
 
 
-@with_unsupported_dtypes({"2.1.2 and below": ("float16", "complex")}, "torch")
+@with_unsupported_dtypes({"2.1.0 and below": ("float16", "complex")}, "torch")
+@with_unsupported_dtypes({"2.5.1 and below": ("bfloat16",)}, "paddle")
```
Comment (vedpatwardhan) [inline ivy/functional/frontends/torch/pointwise_ops.py:185]: Hey @yopknopixx, actually the use of the `with_unsupported_dtypes` decorator in the frontends primarily refers to the datatypes that the function in the actual framework that the frontend belongs to doesn't support. For specifying backend-specific unsupported dtypes, we should update the corresponding decorator in the backend implementation (e.g. the decorator in the `paddle` backend implementation of `ivy.erf`), same applies to `ivy.log`. Thanks 😄

**Review by vedpatwardhan — CHANGES_REQUESTED** [review]

**Comment by yopknopixx** [comment]
This seems to gave been fixed on `main`. Closing the PR. Thanks for the review @vedpatwardhan! 😊

**Pull request closed without merging**